smallrye/smallrye-mutiny
# Add more tests for some Multi operators

Several Multi operators where missing tests.
This PR adds some and remove unused code.

- Rename UniOnFailureMApToTest to UniOnFailureTransformTest
- Remove unused constructor
- Verify the behavior of the Distinct Operator
- Improve Multi Skip Until operator and add more tests
- Verify the filter operator upon cancellation and failures/exceptions
- Add more test to Multi flatmap (transformToMulti)
- Add missing @return in MultiOnEvent javadoc
- Remove unused method in MultiBufferWithTimeoutOp
- Add test for the MutlOnItemTransform operator
- Add tests for Queues and make sure all Queue implementations throw UnsupportedOperationExceptions for unsupported operations
- Verify the behavior of the methods from DrainUtils
- Verify failure collection when merging multis
- Removed unused code in MultiOverflow, MultiSubscribe, and Subscribers
- Rename the UniRunSubscriptionOn operator
